### PR TITLE
Display comment pop-up and features

### PR DESCRIPTION
**Pull request by Grandi0z**
## This pull request adds a pop-up with different features : 
- Displays a Pokemon image
- Displays comment for the Pokemon
- Adds comments and comment counter
## Change made : 
- Add buildCommentPopUp.js. A class that built the pop-up
- Add comment.js file which creates a comment object
- Add involevementAp.js which contains API functions
- Add styles in the index.css
- Call builCommentPopUp in pokemon.js

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -143,3 +155,125 @@ footer {
...
+  right: 0.5rem;
+}
+
+.pop-up-buttons:focus {
+  color: red;
+}
+
+.comment-pop-up-image-container {
+  height: 45vh;
+}
```
Comment (rudicarrilloypr) [inline src/index.css]: Captura de pantalla 2023-07-05 a la(s) 6 56 21

- the pokemon title name it's showing a little bit over the Pokemon icon, maybe adding some padding between them can help to fix the pop-up window style.

File: src/assets/logo/pokemon-logo.png
Comment (rudicarrilloypr) [inline src/assets/logo/pokemon-logo.png:1]: Captura de pantalla 2023-07-05 a la(s) 6 45 36

the pokemon icon is not showing, I'm not sure what changed, let me know if your computer it's showing the logo, please

**Review by rudicarrilloypr — COMMENTED** [review]
> Hi @Grandi0z   ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next task but we are almost there!

 
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used Gitflow ✔️
> * Perfect directory distribution ✔️
>
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._


Captura de pantalla 2023-07-05 a la(s) 6 45 48
>

- [x] for some reason the README file I added isn't on the updated project, my theory is that maybe when I last committed the file you already did your own pull to your local and didn't get the latest commit from myself with the README file updated, I can sent you the README file on slack for you to add it on your new commit.

File: src/index.css (hunk 1 above)
Comment (Grandi0z) [reply src/index.css]: Oh. thanks, I hadn't noticed that. It was working.
Apparently, I made a change in the CSS before making the pull request. I will fix it, thank you

File: src/assets/logo/pokemon-logo.png
Comment (Grandi0z) [reply src/assets/logo/pokemon-logo.png:1]: I have no idea how the logo disappeared.
I don't even see it in my logo folder anymore. Could you please send it back to me?

**Review by rudicarrilloypr — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 


Hi @Grandi0z , 

Your project tasks are complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 
-  Good README file. :heavy_check_mark: 
-  Correct GitFlow. :heavy_check_mark: 
- "add new comment" form on the Comments pop up according to the wireframe
- When the popup loads, the web app retrieves data from the API to show the item comments.
- Comments counter on the popup

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@rudicarrilloypr) in your question so I can receive the notification.

**Pull request merged**